libsolidity/interface/ABI.cpp

```diff
@@ -36,8 +36,6 @@ Json::Value ABI::generate(ContractDefinition const& _contractDef)
 		Json::Value method;
 		method["type"] = "function";
 		method["name"] = it.second->declaration().name();
-		// TODO: deprecate constant in a future release
-		method["constant"] = externalFunctionType->stateMutability() == StateMutability::Pure || it.second->stateMutability() == StateMutability::View;
```

Comment:
We should put this warning into the documentation saying it will be removed with 0.5.0.

Reply:
Done.